doc/geany.txt

```diff
@@ -4580,14 +4580,14 @@ line_wrap_indent
 
 translucency
     Translucency for the current line (first argument) and the selection
-    (second argument). Values between 0 and 256 are accepted.
+    (second argument). Values between 0 and 255 are accepted.
 
     Note for Windows 95, 98 and ME users:
-    keep this value at 256 to disable translucency otherwise Geany might crash.
+    keep this value at 255 to disable translucency otherwise Geany might crash.
```

Comment:
We could probably remove this whole note since AFAIK we don't support anything older than Windows XP (if even that).